commands/migratelxc.go

```diff
@@ -346,3 +370,211 @@ func renameLXDContainers(
 	}
 	return group.Wait()
 }
+
+// startLXDContainers starts the LXD containers that have
+// just been migrated from LXC, or were already migrated
+// but not yet started.
+func startLXDContainers(
+	lxcByHost map[*state.Machine][]*state.Machine,
+	lxdByHost map[*state.Machine]map[string]*lxdContainer,
+	containerNames map[*state.Machine]containerNames,
+) error {
+	running := make(map[string]bool) // keyed by LXD container name
+	for _, containers := range lxdByHost {
+		for name, container := range containers {
+			if container.IsActive() {
+				running[name] = true
+			}
+		}
+	}
+	var group errgroup.Group
+	for host, containers := range lxcByHost {
```

Comment:
What about containers that were already migrated and renamed but not yet restarted? Oh, they'll still be in `lxcByHost` since that's populated from state, not from the actual machines, right?